server/api/utils/dateAndTimeUtils.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * @description Convert date string to string compatible with insertion into postgres's make_timestampz function.
+ * @params date (string) formated 'yyyy-mm-dd'
+ * @returns object containing year month day.
+ */
+
+const convertDateToTimestampz = (date) => {
+  const dateObject = {}
+  const str = date.split('-');
+  ['year', 'month', 'day'].map((key, idx) => dateObject[key] = str[idx]);
+  return dateObject;
```

Comment:
hmm I think you could simplify this with some array destructuring:

```suggestion
  const [year, month, day] = date.split('-');
  return {year, month, day};
```